src/ensembl/src/shared/accordion/Accordion/Accordion.test.tsx

```diff
@@ -0,0 +1,222 @@
+import { mount, ReactWrapper } from 'enzyme';
+import * as React from 'react';
```

Comment:
Due to our typescript settings, `import React from 'react'` works just fine; no need to import star.

Reply:
That's done.